extensions/amp-brid-player/0.1/amp-brid-player.js

```diff
@@ -268,6 +273,50 @@ class AmpBridPlayer extends AMP.BaseElement {
     });
   }
 
+  /**
+   * Requests consent data from consent module
+   * and forwards information to player
+   * @private
+   */
+  getConsentData_() {
+    const consentPolicy = super.getConsentPolicy() || 'default';
+    const consentStatePromise = consentPolicy
+      ? getConsentPolicyState(this.element, consentPolicy)
+      : Promise.resolve(null);
+    const consentStringPromise = consentPolicy
+      ? getConsentPolicyInfo(this.element, consentPolicy)
+      : Promise.resolve(null);
```

Comment:
You always have a `consentPolicy` since you're falling back to `'default'`.

```suggestion
    const consentPolicy = this.getConsentPolicy() || 'default';
    const consentStatePromise = getConsentPolicyState(this.element, consentPolicy);
    const consentStringPromise = getConsentPolicyInfo(this.element, consentPolicy);
```

Reply:
Thank you for taking a look @alanorozco , this change is implemented.

Reply:
Thank you for taking a look @alanorozco , this change is implemented.